File: SaasFeeGuides/SaasFeeGuides/Models/CustomerBooking.cs
using System;

namespace SaasFeeGuides.Models
{
    public class CustomerBooking
    {
        public int? Id { get; set; }
        public int ActivityDateSkuId { get; set; }
        public string ActivitySkuName { get; set; }
        public DateTime DateTime { get; set; }
        public string CustomerEmail { get; set; }
     
        public int NumPersons { get; set; }
        public double PriceAgreed { get; set; }

        public bool HasPaid { get; set; }
        public bool HasConfirmed { get; set; }
        public bool HasCancelled { get; set; }
        public string CustomerDisplayName { get; set; }
        public string CustomerNotes { get; set; }
        public string PriceOptionName { get; internal set; }
    }
}

File: SaasFeeGuides/SaasFeeGuides/ClientApp/src/app/viewModels/customerBooking.ts
export class CustomerBooking {

  constructor(
    public customerDisplayName : string,
    public activitySkuName : string,
    public dateTime: Date,
    public customerEmail: string,     
    public numPersons: number,
    public hasPaid: boolean,
    public hasConfirmed: boolean,
    public priceOptionName:string) { }

}
